featuretools/utils/recommend_primitives.py

```diff
@@ -0,0 +1,273 @@
+from typing import List
+
+import pandas as pd
+
+from featuretools.computational_backends import calculate_feature_matrix
+from featuretools.entityset import EntitySet
+from featuretools.primitives.utils import get_transform_primitives
+from featuretools.synthesis import dfs, get_valid_primitives
+
+ORDERED_PRIMITIVES = [  # primitives that require ordering
+    "absolute_diff",
+    "cum_sum",
+    "cum_count",
+    "cum_mean",
+    "cum_max",
+    "cum_min",
+    "cumulative_time_since_last_false",
+    "cumulative_time_since_last_true",
+    "diff",
+    "diff_datetime",
+    "exponential_weighted_average",
+    "exponential_weighted_std",
+    "exponential_weighted_variance",
+    "greater_than_previous",
+    "is_first_occurrence",
+    "is_last_occurrence",
+    "is_max_so_far",
+    "is_min_so_far",
+    "lag",
+    "less_than_previous",
+    "percent_change",
+    "same_as_previous",
+    "time_since_previous",
+]
+
+
+DEPRECATED_PRIMITIVES = [
+    "multiply_boolean",  # functionality duplicated by 'and' primitive
+    "numeric_lag",  # deperecated and replaced with `lag`
+]
+
+REQUIRED_INPUT_PRIMITIVES = [
+    "count_string",
+    "distance_to_holiday",
+    "is_in_geobox",
+    "score_percentile",
+    "subtract_numeric_scalar",
+    "scalar_subtract_numeric_feature",
+    "not_equal_scalar",
+    "multiply_numeric_scalar",
+    "modulo_numeric_scalar",
+    "divide_numeric_scalar",
+    "add_numeric_scalar",
+    "equal_scalar",
+    "greater_than_equal_to_scalar",
+    "less_than_equal_to_scalar",
+    "divide_by_feature",
+    "greater_than_scalar",
+    "less_than_scalar",
+    "modulo_by_feature",
+    "time_since",
+    "savgol_filter",
+    "isin",
+    "numeric_bin",
+]
+
+OTHER_PRIMITIVES_TO_EXCLUDE = [  # Excluding all multi-input primitives and numeric primitives that don't handle skew
+    "absolute",
+    "negate",
+    "not",
+    "and",
+    "or",
+    "is_zero",
+    "is_null",
+    "greater_than_equal_to",
+    "equal",
+    "greater_than",
+    "is_whole_number",
+    "not_equal",
+    "multiply_numeric",
+    "multiply_numeric_boolean",
+    "add_numeric",
+    "less_than_equal_to",
+    "divide_numeric",
+    "less_than",
+    "subtract_numeric",
+    "modulo_numeric",
+    "cosine",
+    "tangent",
+    "sine",
+]
+
+DEFAULT_EXCLUDED_PRIMITIVES = (
+    REQUIRED_INPUT_PRIMITIVES
+    + DEPRECATED_PRIMITIVES
+    + ORDERED_PRIMITIVES
+    + OTHER_PRIMITIVES_TO_EXCLUDE
+)
+
+# TODO: Make this list more dynamic
+TIME_SERIES_PRIMITIVES = [
+    "expanding_count",
```

Comment:
@ozzieD @thehomebrewnerd 
Should exponential primitives be included here?

Reply:
No - those primitives are not created properly to be used for time series problems where a measurement delay might be present.